### ISSUE TITLE
Collection description rejects images

### ISSUE THREAD

**Issue by pdxrlk** [outline/outline#2366]
The description field for collections allows you to enter an arbitrary block. I think it assumes text will be used, but it allows insertion of any block including images.  However when I insert an image file, it shows up for a second and then disappears.

Honestly, I'd rather that a collection page just be editable like any other document, with the document list on the bottom (like nested documents is handled).

**To Reproduce**

1. Go to a collection
2. Enter an image into the document description field
3. Notice that the image appears briefly, and then disappears

**Expected behavior**
expected the image to just display like it would on any other page.  I also expected the description field to expand to accommodate whatever I enter into the collection description.

**Screenshots**

**Outline (please complete the following information):**
- Install: getoutline.com 

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Safari

**Comment by tommoor** [comment]
I've not been able to reproduce this issue unfortunately, a variety of images are uploading and being persisted correctly for me.

It could be the image type or some other factor. If you're able to open developer tools and see the network response this might give some clue as to what's happening

**Comment by pdxrlk** [comment]
Strange. I've tried dragging/dropping and using /image (specifically within Collection description fields) to no avail.  I've tried Safari and Chrome.

Attached is a recording with the network timeline exposed.  
p.s. it seems odd that I can edit the collection without actually choosing "edit" from the ellipsis menu (and I can't actually edit the description when I am in edit mode)

https://user-images.githubusercontent.com/15617411/127056482-cd6897b0-b702-402e-8738-ac5b0ffc56f3.mov
